fastlane/metadata/fr-FR/keywords.txt

```diff
@@ -0,0 +1 @@
+numérisation, documents, rapide, simple, iOS, sauvegarder, partager, photos, images, PDF, email, SMS, reçus, notes, factures, cartes de visite, certificats
```

Comment:
This file has more than 100 characters. Please keep the most important keywords and remove others to reach the limit of 100 characters.